source/class/cv/Application.js

```diff
@@ -307,9 +307,27 @@ qx.Class.define("cv.Application",
           needsConfirmation: false
         }
       );
+      if (qx.core.Environment.get('cv.sentry') && window.Sentry) {
+        notification.actions.link.push(
+          {
+            title: qx.locale.Manager.tr("Send error to sentry.io"),
+            action: function () {
+              Sentry.captureException(ex);
+            },
+            needsConfirmation: false
+          }
+        );
+      }
       cv.core.notifications.Router.dispatchMessage(notification.topic, notification);
     },
 
+    throwError: qx.core.Environment.select('qx.globalErrorHandling', {
+      "true":  function () {
+        window.onerror(new Error('test error'));
```

Comment:
Is "test error" intended or a left over?

Reply:
No this one is intented to have an easy way to test the globalErrorHandling (when its enabled)